Cannot fetch eu.simonbinder:sqlite3-native-library:3.32.3.

Here is the output of `flutter build apk`:

```

Building without sound null safety
For more information see https://dart.dev/null-safety/unsound-null-safety

                                                                        
FAILURE: Build failed with an exception.                                
                                                                        
* What went wrong:                                                      
Could not determine the dependencies of task ':sqlite3_flutter_libs:compileReleaseAidl'.
> Could not resolve all task dependencies for configuration ':sqlite3_flutter_libs:releaseCompileClasspath'.
   > Could not resolve eu.simonbinder:sqlite3-native-library:3.32.3.    
     Required by:                                                       
         project :sqlite3_flutter_libs                                  
      > Could not resolve eu.simonbinder:sqlite3-native-library:3.32.3. 
         > Could not get resource 'https://dl.bintray.com/sbinder/sqlite3-native-library/eu/simonbinder/sqlite3-native-library/3.32.3/sqlite3-native-library-3.32.3.pom'.
            > Could not GET 'https://dl.bintray.com/sbinder/sqlite3-native-library/eu/simonbinder/sqlite3-native-library/3.32.3/sqlite3-native-library-3.32.3.pom'. Received status code 403 from server: Forbidden
                                                                        
* Try:                                                                  
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
                                                                        
* Get more help at https://help.gradle.org                              
                                                                        
BUILD FAILED in 11s                                                     
Running Gradle task 'assembleRelease'...                                
Running Gradle task 'assembleRelease'... Done                      12,1s
Gradle task assembleRelease failed with exit code 1
```

```
$ flutter --version
Flutter 2.0.6 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 1d9032c7e1 (12 days ago) • 2021-04-29 17:37:58 -0700
Engine • revision 05e680e202
Tools • Dart 2.12.3
$ java --version
openjdk 11.0.10 2021-01-19
OpenJDK Runtime Environment (build 11.0.10+9-suse-2.2-x8664)
OpenJDK 64-Bit Server VM (build 11.0.10+9-suse-2.2-x8664, mixed mode)
$ git rev-parse HEAD
a3eafe1b38149663b9b5aa29abed687a4983f4f3
```

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -68,7 +68,7 @@ dependencies:
       # url: https://github.com/ibrierley/flutter_map_line_editor.git  
 
   # MIT
-  sqlite3_flutter_libs: ^0.3.0
+  sqlite3_flutter_libs: ^0.4.0
 
   # BSD
   flutter_map: ^0.10.1

